### PR TITLE
Iris Fixes

### LINKED ISSUE #11: `.idea/vcs.xml` is deleted when running the idea import task
This causes the Commit & Pull Requests tab to disappear, and I have to manually go into VCS -> Enable Version Control Integration to restore them.

### LINKED ISSUE #12: Bootstrap does not trim / validate tab-separated elements when parsing brachyurabootstrapconf.txt
This line will not be parsed correctly:

```
https://repo.eclipse.org/content/groups/releases/org/eclipse/jgit/org.eclipse.jgit/6.0.0.202111291000-r/org.eclipse.jgit-6.0.0.202111291000-r.jar	a6184d0441ad4a912f73c2f9e14f2fe3826b9306	org.eclipse.jgit-6.0.0.202111291000-r.jar	 true
```

(note the extra space before `true`)

### LINKED ISSUE #13: Cannot easily create a new ProcessorChain by extending an existing ProcessorChain with additional elements
If this were possible it would be easier to add on additional processors in the resourcesProcessingChain by calling super & extending the result with any extra desired processors.

### PR DESCRIPTION
**Pull request by coderbot16**
- Bumps `brachyura` to 0.57
- Bumps `brachyura-mixin-compile-extensions` to 0.5
- Fixes https://github.com/CoolCrabs/brachyura/issues/12
- Fixes https://github.com/CoolCrabs/brachyura/issues/11
- Fixes https://github.com/CoolCrabs/brachyura/issues/13
- Hacky fix to suppress "WARN: Unable to find byte.class"
- Makes the autorelease setup automatically work on forked repositories
- Fixes a critical method remapping bug in brachyura-mixin-compile-extensions

### CODE REVIEW

File: brachyura/src/main/java/io/github/coolcrabs/brachyura/ide/Intellijank.java [hunk 1]
```diff
@@ -49,7 +49,27 @@ public void updateProject(Path projectDir, IdeProject ideProject) {
...
+                        // otherwise we'll reset people's Git configurations
+                    } else {
+                        // delete other files
+                        try {
+                            Files.delete(path);
+                        } catch (IOException e) {
+                            Util.sneak(e);
+                        }
+                    }
+                });
```
Comment (Chocohead) [inline brachyura/src/main/java/io/github/coolcrabs/brachyura/ide/Intellijank.java]: If you're deleting everything but `vcs.xml` I think you could shorten this to
```java
try (DirectoryStream<Path> files = Files.newDirectoryStream(ideaPath,
		file -> !"vcs.xml".equals(file.getFileName().toString()))) {
	for (Path file : files) {
		if (Files.isDirectory(file)) {
			PathUtil.deleteDirectory(file);
		} else {
			Files.delete(file);
		}
	}
}
```

Comment (coderbot16) [reply brachyura/src/main/java/io/github/coolcrabs/brachyura/ide/Intellijank.java]: Thanks for the suggestion, will test this out later.
